helm/cluster-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,35 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "cluster-operator.name" -}}
+{{- .Chart.Name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "cluster-operator.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+Common labels
+*/}}
+{{- define "cluster-operator.labels" -}}
+app: {{ include "cluster-operator.name" . | quote }}
+app.giantswarm.io/branch: {{ .Values.project.branch | quote }}
+app.giantswarm.io/commit: {{ .Values.project.commit | quote }}
+{{ include "cluster-operator.selectorLabels" . }}
```

Comment:
I wouldn't go as crazy to sort `include` contents :D Just put it on the top or the bottom :)

Reply:
Well, I did :sweat_smile: 
Ok